src/utilities/formInputConfigs.js

```diff
@@ -116,6 +117,22 @@ const FORM_INPUT_CONFIGS = {
     isRequired: false,
     label: 'Address 2:',
   },
+  [FORM_INPUT_KEYS.REGISTRATION_CODE]: {
+    type: 'text',
+    initialValue: '',
+    key: 'registrationCode',
+    validator: input => input.length < 50,
+    isRequired: false,
+    label: 'Registration code:',
+  },
+  [FORM_INPUT_KEYS.REGISTRATION_CODE]: {
```

Comment:
why are there two REGISTRATION_CODEs?
are they required or not?

Reply:
🤣 

Not in msupply!

side note: It is required in mSupply that you enter a `first name` and a `last name`, but also `initials`. I am in fact not a doctor so I don't have domain expertise here, but that sure does seem odd

fa5f84ad1138157bb7c9b5bf6a6d39cc921693d7